lib/components/input-group.vue

```diff
@@ -1,19 +1,25 @@
 <template>
-    <component :is="tag" :class="classObject" role="group">
+    <component :is="tag" :id="id || null" :class="classObject" role="group">
```

Comment:
It seems potentially beneficial to have that prop warning in place since I can't think of a good reason for issuing an element id as an empty string.